docs/octopus-cloud/static-ip.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: Static IP address
+title: Static IP Address
```

Comment:
Sorry about this one. I'm in the process of changing the titles in the docs to sentence case to be consistent with the rest of octopus.com:
```suggestion
title: Static IP address
```